builtin/fstk/ui.lua

```diff
@@ -54,29 +54,42 @@ end
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 
+local function wordwrap_hack(str)
```

Comment:
because formspecs should support wrapping themselves already. Also I've only moved the code. can you discuss this with @rubenwardy ?